package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever-components",
-  "version": "0.7.3",
+  "version": "0.8.0",
```

Comment:
yeah - IIRC, a pre-1.0 minor bump is equivalent to a major version bump. npm won't automatically pick up 0.8.0 for `~0.7.x`, so clients would have to explicitly update.

currently only used in the apps dashboard, where we're already using column ids instead of indices